tests/unit/test_file_handle_cache.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tests the file handle cache
+"""
+
+from ga4gh import datamodel
+
+import unittest
+import tempfile
+import os
+
+import ga4gh.frontend as frontend
+
+
+class TestFileHandleCache(datamodel.PysamFileHandleCache, unittest.TestCase):
```

Comment:
Ah, I see, we're doing a white-box test here... I think it would be better to add a public method like `getCachedFiles` or something to the `FileHandleCache` which returns the list of file names that are currently in the cache. This will let us test the external functionality without needing to access the internal state of the object, and coupling our test cases with the implementation.